integration-tests/src/lib.rs

```diff
@@ -139,5 +148,55 @@ pub struct CompiledContract {
     pub bin_runtime: Bytes,
 }
 
+/// Build circuit input builder for a block
+pub async fn build_circuit_input_builder_block(block_num: u64) {
```

Comment:
It's to move a fn from ```tests/circuit_input_builder.rs``` to ```src/lib.rs```.  It's cleaner to in ```src/lib.rs``` since it's used by two integration tests.